pkg/metrics/server/setup.go

```diff
@@ -130,6 +133,14 @@ func PortFromEnv() int {
 	return port
 }
 
+func IPFromEnv() string {
```

Comment:
please rename `AddressFromEnv` for consistency

Reply:
On it.